core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -106,6 +131,62 @@ public void testImportMutationsFile() throws Exception {
         ImportProfileData secondRunner = new ImportProfileData(secondArgs);
         secondRunner.run();
         validateMutationAminoAcid (geneticProfileId, secondSampleId, 2842, "L113P");
+        // assume a MUTATION_COUNT record has been added for the sample and the
+        // count is 1, the other one is a germline mutation
+        // also confirm mutation count for first sample is still correct
+        clinicalData = DaoClinicalData.getSampleData(study.getInternalId(), new ArrayList<String>(Arrays.asList("TCGA-AA-3664-01", "TCGA-AA-3665-01")), clinicalAttribute);
+        assert(clinicalData.size() == 2);
+        assertEquals("8", clinicalData.get(0).getAttrVal());
+        assertEquals("1", clinicalData.get(1).getAttrVal());
+        // remove profile at the end
+        DaoGeneticProfile.deleteGeneticProfile(geneticProfile);
+        assertNull(DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations"));
```

Comment:
this unit test is not checking whether the deleteGeneticProfile() works properly, so we may not need this assert

Reply:
this could be done a bit cleaner by doing this in `@After`, since each test assumes the mutation profile doesn't exist. I do think it's good to make sure there that the profile is completely deleted even though that's not what's being tested. Otherwise it might be p unclear why these tests are failing. It's kinda like using it as an `assume that x` rather than the test itself